libsasl2/plan.sh

```diff
@@ -4,7 +4,7 @@ pkg_origin=core
 pkg_version="2.1.26"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=("custom")
-pkg_source="https://cyrusimap.org/releases/${pkg_distname}-${pkg_version}.tar.gz"
+pkg_source="https://ftp.osuosl.org/pub/blfs/conglomeration/cyrus-sasl/cyrus-sasl-2.1.27.tar.gz"
```

Comment:
@MindNumbing: Is this meant to be the same as for core/cyrus-sasl?  If yes, then the shasum will need to be the same as well.